packages/create-cloudflare/src/templates.ts

```diff
@@ -142,10 +142,18 @@ export type TemplateConfig = {
 	/** The key of the package.json "scripts" entry for previewing the project. Defaults to undefined (there might not be such script) */
 	previewScript?: string;
 
+	/** The file path to the generated types file. Defaults to worker-configuration.d.ts*/
+	typesPath?: string;
+	/** The name of the Env type generated by wrangler types. Defaults to `Env`*/
+	envInterfaceName?: string;
+
 	/** The file path of the template. This is used internally and isn't a user facing config value.*/
 	path?: string;
 
 	bindings?: Record<string, unknown>;
+
+	/** Default false. To accomodate SSG frameworks etc. */
```

Comment:
```suggestion
	/** Defaults to `"none"`. To accomodate SSG frameworks etc. */
```

Comments:
- Maybe we can quickly document each value?
- I'm not sure about what "To accomodate SSG frameworks etc." means here, do we need to keep this

Reply:
IIUC the default is actually "generated"